CHANGELOG.md

```diff
@@ -3,6 +3,78 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.15.1
+_Sep 28, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧰 Fix header filters menu auto closing on render (#10483) @MBilalShafi
```

Comment:
Maybe you have ideas on how to improve the wording here?
cc @noraleonte
```suggestion
- 💫 Add customization playground (#9581) @noraleonte

  You can play around with style customization options on the [Date Picker documentation](https://mui.com/x/react-date-pickers/date-picker/#customization).
  We are thrilled to hear your feedback about this functionality!

- 🧰 Fix header filters menu auto closing on render (#10483) @MBilalShafi
```

Reply:
> Add customization playground for the `DatePicker`

Maybe add in the title that it's for the pickers (or the DatePicker), so that grid users can just ignore it without diving deeper
I guess it's a general rule, reading the 1st line should always make clear which package we are talking about

Reply:
LGTM! 💯 Thanks for including this! I agree with Flavien that we should mention that it's for the pickers 👍

Reply:
I've updated the suggestion to be more clear. 👌

Reply:
IMO wording is fine and transports a excitement! 🚀 
WDYT @noraleonte ?